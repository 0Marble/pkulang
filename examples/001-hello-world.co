% main() void: entry point
fn main() void {
	print("Enter your name: ");
	let name: str = read_line();
	% print can take many args
	println("Hello,", name, "!");
}

