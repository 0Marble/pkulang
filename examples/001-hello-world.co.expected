Enter your name: Hello, alex!
